gulpfile.js

```diff
@@ -103,8 +103,11 @@ gulp.task('copy:main.css', function () {
 
     return gulp.src(dirs.src + '/css/main.css')
                .pipe(plugins.header(banner))
-               .pipe(gulp.dest(dirs.dist + '/css'));
-
+                .pipe(plugins.autoprefixer({
+                    browsers: ['last 2 versions', 'ie >= 8'],
```

Comment:
The browser support for H5BP is Chrome, Firefox, IE8+, Opera, and Safari, correct?
[Autoprefixer](https://github.com/postcss/autoprefixer#browsers) uses Browerlist to [specify support configuration](https://github.com/ai/browserslist/blob/master/README.md).

The configuration I've added...
```javascript
browsers: ['last 2 versions', 'ie >= 8'],
```
...means the last 2 versions for each major browser, and IE8+. That should cover it, however, you can be more specific in the configuration, e.g.:
```javascript
browsers: ['Chrome', 'Firefox', 'ie 8', 'ie 9', 'ie 10', 'ie 11', 'Opera', 'Safari'],
```